File: functions/updated_hello_world.c
//Lila Shearer, Updated Hello World C
#include <stdio.h>


void greeting(char answer[50]){
    
    printf("Hello, %s!\n", answer);

}

int main(void){
    printf("Welcome to updated Hello World. This program is designed to say hello to names. You will get the opportunity to insert a name and see other names that are greeted.\n");
    char answer[50];
    printf("Please type a name:\n");
    scanf("%s", answer);
    greeting(answer);
    greeting("Sally");
    greeting("Alexander");
    greeting("Jacob");
    greeting("Zachary");
    return 0;
}

File: functions/updated_hello_world.py
# Lila Shearer, Updated Hello World
print("Welcome to updated Hello World! This program takes in 5 names and says hellos to them individually.\n")
def name():
    name_input = input("Please write a name:\n").strip().capitalize()
    return (f"Hello, {name_input}!")
    
print(name())
print(name())
print(name())
print(name())
print(name())

#I did user inputs for this, but in case that doesn't fit the assignment requirements, here is what it looks like if it's not user inputs:
#def name(nameVariable):
#    return (f"Hello, {nameVariable}!")
#
#print(name("Peter"))
#print(name("John"))
#print(name("Jacob"))
#print(name("Emily"))
#print(name("Charlie"))